packages/subapp-react/test/spec/ssr-framework.spec.js

```diff
@@ -1,15 +1,36 @@
 "use strict";
-
 const url = require("url");
 const React = require("react"); // eslint-disable-line
 const lib = require("../../lib");
 const { withRouter } = require("react-router");
 const { Route, Switch } = require("react-router-dom"); // eslint-disable-line
-const { asyncVerify } = require("run-verify");
 const Redux = require("redux");
 const { connect } = require("react-redux");
+const { Stream } = require("stream");
 
 describe("SSR React framework", function () {
+  function getTestWritable() {
+    const writable = new Stream.PassThrough();
+    writable.setEncoding("utf8");
+    const output = { result: "", error: undefined };
+    writable.on("data", chunk => {
+      output.result += chunk;
+    });
+    writable.on("error", error => {
+      output.error = error;
+    });
+    const completed = new Promise(resolve => {
+      writable.on("finish", () => {
+        resolve();
+      });
+      writable.on("error", () => {
+        resolve();
+      });
+    });
+    return { writable, completed, output };
+  }
+  const { writable, output, completed } = getTestWritable();
```

Comment:
Major: this should be in the context of each it block, otherwise it is using the same instance of `writable` for every `output`. The tests are currently unreliable. I recommend varying up the text props in each test simply to ensure reliability in the future. 

To see the problem, change the text props of the first test using `writable` and `output`. You will see that all following tests will use the same text props.